api/envoy/extensions/access_loggers/grpc/v3/als.proto

```diff
@@ -101,4 +102,20 @@ message CommonGrpcAccessLogConfig {
 
   // A list of custom tags with unique tag name to create tags for the logs.
   repeated type.tracing.v3.CustomTag custom_tags = 8;
+
+  // Define the log condition for critical access logs.
+  // Logs that match the filter are not sent to `StreamAccessLogs`,
+  // but are sent to `CriticalAccessLogs`.
+  config.accesslog.v3.AccessLogFilter critical_buffer_log_filter = 9;
```

Comment:
Does it mean we only use reliable way to send critical logs? Or the "critical log" means "reliable log"?

Reply:
Yes. "critical log" and "reliable log" have the same meaning.

Reply:
It would be better to use "reliable log" as **critical** is truly a common log level.

Reply:
I agree that it would be better to use `reliable` consistently.